readme.md

```diff
@@ -152,6 +182,30 @@ Gets the latest version of all browsers.
   * `running_sessions`: Number of running sessions.
   * `sessions_limit`: Number of allowable concurrent sessions.
 
+### Screenshots API
+
+#### BrowserStack.createScreenshotClient( settings )
+
+Creates a new client instance.
+
+* `settings`: A hash of settings that apply to all requests for the new client.
+  * `username`: The username for the BrowserStack account.
+  * `password`: The password for the BrowserStack account.
+
+#### screenshotClient.getBrowsers( callback )
+
+Gets the list of available browsers.
+
+* `callback` (`function( error, browsers )`): A callback to invoke when the API call is complete.
+  * `browsers`: An array of [browser objects](#browser-objects).
+
+#### screenshotClient.getScreenshot( options, callback )
+
+Runs a job to take screenshots. Options is an object with the following keys :
```

Comment:
Looks like you never finished documenting this.

Also missing docs for `getJob()`.